plugin/goplugin_rpc.go

```diff
@@ -0,0 +1,80 @@
+// Copyright 2025 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+package plugin
+
+import (
+	"context"
+
+	"github.com/veraison/services/log"
+	"go.uber.org/zap"
+	"google.golang.org/protobuf/types/known/emptypb"
+
+	"github.com/veraison/ratsd/proto/compositor"
+)
+
+type GRPCServer struct {
+	compositor.UnimplementedCompositorServer
+	Impl IPluggable
+}
+
+func (s *GRPCServer) GetSubAttesterID(ctx context.Context, e *emptypb.Empty) (*compositor.SubAttesterIDOut, error) {
+	return s.Impl.GetSubAttesterID(), nil
+}
+
+func (s *GRPCServer) GetSupportedFormats(ctx context.Context, e *emptypb.Empty) (*compositor.SupportedFormatsOut, error) {
+	return s.Impl.GetSupportedFormats(), nil
+}
+
+func (s *GRPCServer) GetEvidence(ctx context.Context, in *compositor.EvidenceIn) (*compositor.EvidenceOut, error) {
+	return s.Impl.GetEvidence(in), nil
+}
+
+type GRPCClient struct {
+	client compositor.CompositorClient
+}
+
+func (c *GRPCClient) GetSubAttesterID() *compositor.SubAttesterIDOut {
+	resp, err := c.client.GetSubAttesterID(context.Background(), &emptypb.Empty{})
+	if err != nil {
+		return &compositor.SubAttesterIDOut{
+			Status: &compositor.Status{Result: true, Error: err.Error()},
```

Comment:
On error, status.result should be false

Reply:
Switched the logic to `Result: false` in the latest revision.